### PR TITLE
refactor(desktop): Removed all timeline related code

### LINKED ISSUE #4064: remove timeline code
this feature is being deprecated we should remove the associated code

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #4064

### What does the PR do
Removed all timeline associated code from base_bc.
This feature is deprecated and shall be removed

### Affected areas
Timeline

Depends on https://github.com/status-im/status-go/pull/2451 and https://github.com/status-im/status-lib/pull/122

### CODE REVIEW

File: nim_status_client.log [hunk 1]
```diff
@@ -0,0 +1 @@
+INF 2021-12-01 22:08:22.930+02:00 Terminating the app as the second instance topics="main" tid=953089 file=nim_status_client.nim:73
```
Comment (alaibe) [inline nim_status_client.log:1]: nit: do not commit, maybe add to gitignore

**Review by alaibe — APPROVED** [review]
Nice one!

**Review by PascalPrecht — APPROVED** [review]
Amazing @alexandraB99 Great job!

As @alaibe said, the log file should be removed from this commit tho

File: nim_status_client.log (hunk 1 above)
Comment (alexandraB99) [reply nim_status_client.log]: meh..removed and added in .gitignore

**Comment by alexandraB99** [comment]
@PascalPrecht @alaibe , removed log file and also updated .gitignore. Please let me know if that's ok in this commit or we should do it separately!

**Comment by PascalPrecht** [comment]
Looks good! Missing colon in the message, but looks good! 😂

**Review by PascalPrecht — APPROVED** [review]

**Pull request merged**